app/players/elves/elf_ranger.py

```diff
@@ -0,0 +1,22 @@
+from app.players.elves.elf import Elf
+
+
+class ElfRanger(Elf):
+    def __init__(self,
+                 nickname: str,
+                 musical_instrument: str,
+                 bow_level: int
+                 ) -> None:
+        super().__init__(nickname, musical_instrument)
+        self._bow_level = bow_level
+
+    def play_elf_song(self) -> None:
+        print(f"{self.nickname} is playing a song"
+              f" on the {self._musical_instrument}")
```

Comment:
`DRY` principle should be followed

Reply:
not fixed yet

Reply:
DRY means don't repeat yourself. Method shouldn't be duplicated